libr/core/cmd_print.c

```diff
@@ -3333,6 +3333,67 @@ static bool cmd_print_blocks(RCore *core, const char *input) {
 	return result;
 }
 
+static bool cmd_pse(RCore *core, const char *input) {
+	if (input[2] == ' ') {
+		const char *fname = r_str_trim_head_ro (input + 3);
+		char *tmp = r_str_newf (R_JOIN_2_PATHS (R_JOIN_2_PATHS (R2_HOME_DATADIR, "util"), "%s"), fname);//todo: add R2_HOME_UTIL
+		char *home = r_str_home (tmp);
+		//free (tmp);
+		tmp = r_str_newf (R_JOIN_2_PATHS (R_JOIN_2_PATHS (R2_SDB, "util"), "%s"), r_str_newf (R_JOIN_2_PATHS ("d", "%s"), fname));
+		char *path = r_str_r2_prefix (tmp);
+		if (!sdb_query_file (core->anal->charset_db, path)) {
```

Comment:
better to just load. using query can rsult on slower loading times and unnecessary features for just k=v

Reply:
use something like this should be done after cconfig.c when changing a specific config var (feel frree to propose one name for it).
```c
char *path = system_charset_path();
int a = sdb_open (path);
char *path = home_charset_path();
int a = sdb_open (path);
```
note that right now sdb_open will create an empty database in case the file doesnt exist, so better use r_file_exists instead before calling sdb_open once

Reply:
this sdb_open issue should be addressed in sdb repo

Reply:
is this resolved?